qa-tests-backend/src/test/groovy/BaseSpecification.groovy

```diff
@@ -98,9 +98,9 @@ class BaseSpecification extends Specification {
 
         orchestrator.createNamespace(Constants.ORCHESTRATOR_NAMESPACE)
 
-        addStackroxImagePullSecret()
-        addGCRImagePullSecret()
-        addRedHatImagePullSecret()
+        addStackroxImagePullSecret(orchestrator)
```

Comment:
How about making orchestrator a field so we can use it from any test

Reply:
I feel like that's a separate change, because even if it would simplify this it would mean we have to update all tests that instantiate an orchestrator to not do it and use the existing one. Would it be fine to leave this as-is and consider your improvement idea separately, perhaps a small papercut?